core/src/main/web/app/helpers/helper.js

```diff
@@ -673,6 +673,9 @@
           return [];
         }
       },
+      hasIPythonEvaluatorsLoaded: function () {
+        return _.intersection(this.getEvaluators(), ['IPython', 'IRuby', 'Julia', 'Python3', 'Torch']).length > 0;
```

Comment:
there has to be a better way...
how about these plugins set a global, then they can just check if the global is set?